packages/typescript/preact-ai-devtools/README.md

```
@@ -0,0 +1,101 @@
+<div align="center">
+  <img src="./media/header_ai.png" >
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)